JPS_VIRTUALSENSOR/VirtualSensorAgent/src/main/java/com/cmclinnovations/virtualsensor/QueryClient.java

```diff
@@ -271,9 +271,8 @@ public void updateStationUsingDispersionRaster(Instant latestTime, Map<String, S
                         double concentration = result.getDouble("val");
                         concentrations.add(concentration);
                     } else {
-                        String errmsg = "Could not obtain raster value";
-                        LOGGER.error(errmsg);
-                        throw new RuntimeException(errmsg);
+                        LOGGER.warn("Could not obtain raster value for {}", rasterFileName);
```

Comment:
What is the effect of this changed? Instead of non-existent it returns null?

Reply:
The concentration value at that time point will be null if raster data cannot be retrieved